projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts

```diff
@@ -9,13 +9,15 @@ import {
 import { MatSnackBar } from '@angular/material/snack-bar';
 import {
     ApiResponseError,
+    DeleteValue,
     IHasPropertyWithPropertyDefinition,
     KnoraApiConnection,
     PropertyDefinition,
     ReadResource,
     ReadValue,
     SystemPropertyDefinition
 } from '@dasch-swiss/dsp-js';
+import { BaseValue } from '@dasch-swiss/dsp-js/src/models/v2/resources/values/base-value';
```

Comment:
wrong import path (see previous comment) - it causes a warning in the terminal when you run `ng s`

Reply:
done in 68cbdf9

Reply:
fixed in c5f29c0